### DISCUSSION TITLE
同源策略阻止评论数统计的请求

### DISCUSSION THREAD

**Discussion by Legend-cpu** [walinejs/waline#1110] [Q&A] [answered]
如题，开启评论数统计时向vercel部署的waline api发送请求时被同源策略阻止了，导致评论数一直加载不出来，因为服务器返回的响应里缺少Access-Control-Allow-Origin字段。我现在加了一点js代码用dom获取了评论数，但就是说，这个问题有什么更好的办法解决么

**Comment by lizheming** [comment, marked as answer]
提供下复现地址看一下，另外 serverURL 最后如果有 `/` 去一下，否则会有问题

**Reply by Legend-cpu** [reply]
好的，把那个`/`去掉之后就没有同源策略阻止的提示了，好怪啊，为什么会这样？
